samples/WebApi/README.md

```diff
@@ -0,0 +1,115 @@
+# Description
+This is a sample application, which uses CoreRT to compile a .NET Core Web API sample.
+TODO WIP - Might not work on Linix
+
+# Building a WebAPI app with CoreRT
+
+## Install the .NET Core SDK
+CoreRT is and AOT-optimized .NET Core runtime. If you're new to .NET Core make sure to visit the [official starting page](http://dotnet.github.io). It will guide you through installing pre-requisites and building your first app.
+If you're already familiar with .NET Core make sure you've [downloaded and installed the .NET Core 2 SDK](https://www.microsoft.com/net/download/core).
+
+## Create your app 
+
+Open a **new** shell/command prompt window and run the following commands.
+``> dotnet new webapi -o myApp`` 
+
+``> cd myApp``
+
+## Add CoreRT to your project
+
+Using CoreRT to compile your application is done via the ILCompiler NuGet package.
+
+In your shell/command prompt navigate to the root directory of your project run the command 
+
+`` dotnet new nuget ``
+
+This will add a nuget.config file to your application. Open the file and in the ``<packageSources> `` element under ``<clear/>`` add the following:
+
+``<add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" /> ``
+``<add key="aspnet-core" value="https://dotnet.myget.org/F/aspnetcore-release/api/v3/index.json" /> ``
+
+Once you've added the package source, add a reference to the compiler by running the following command:
+
+`` dotnet add package Microsoft.DotNet.ILCompiler -v <Version> `` 
+
+where ``<Version>`` is the [latest version of the ILCompiler package](https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.DotNet.ILCompiler) (e.g. 1.0.0-alpha-26006-02).
```

Comment:
If you want the latest prerelease version, you should be able to just do `-v 1.0.0-alpha-*`. No need to hunt down the version on myget manually.